packages/store/project.json

```diff
@@ -23,7 +23,8 @@
       "configurations": {
         "ci": {
           "ci": true,
-          "codeCoverage": true
+          "codeCoverage": true,
+          "verbose": true
```

Comment:
It may be worth leaving the `verbose` in there. Let's see how it changes the ci output...

Reply:
Ok, I removed this, to force a cache invalidation.